services/libs/data-access-layer/src/old/apps/profiles_worker/index.ts

```diff
@@ -58,10 +58,11 @@ export async function runMemberAffiliationsUpdate(
         "organizationId",
         "dateStart",
         "dateEnd",
-        "createdAt"
+        "createdAt",
+        "deletedAt"
       FROM "memberOrganizations"
       WHERE "memberId" = $(memberId)
-        AND "deletedAt" IS NULL
+      AND "deletedAt" IS NULL
```

Comment:
You might want to revert all these changes

Reply:
Done ✅